packages/firestore/test/integration/api/database.test.ts

```diff
@@ -149,6 +150,7 @@ apiDescribe('Database', persistence => {
           expect(docSnapshot.exists).to.be.ok;
           expect(docSnapshot.get('updated')).to.be.false;
           expect(docSnapshot.get('time')).to.be.an.instanceof(Timestamp);
+          expect(docSnapshot.get('nested.time')).to.be.an.instanceof(Timestamp);
```

Comment:
changes to this test seem fine but unrelated to your change?  Are you making it match another platform or something?

Reply:
This is to match the new test `can set server timestamps using mergeFields`.